Source/ACE/Entity/Player.cs

```diff
@@ -254,7 +254,8 @@ public void GrantXp(ulong amount)
             CharacterLevel maxLevel = chart.Levels.Last();
             if (character.Level == maxLevel.Level)
             {
-                character.GrantAdditionalXp(amount);
+                var maxLevelText = new GameMessageSystemChat($"You have already reached max level!", ChatMessageType.Broadcast);
+                Session.WorldSession.EnqueueSend(maxLevelText);

```

Comment:
just comment these lines out or turn into `if (!character.Level == maxLevel.Level)` for now